### ISSUE TITLE
matlabpool doesn't exist in Matlab 2015 and above

### ISSUE THREAD

**Issue by patricksnape** [zhusz/CVPR15-CFSS#3]
Therefore, all the lines where you check `(matlabpool('size')==0)` throw an exception. This can be fixed by replacing them with `isempty(gcp('nocreate'))`.

However, I'm not sure if that call works on older versions of Matlab. If someone confirms this still works then I can make a PR to fix this.

**Comment by zhusz** [comment]
Yes, thanks! And actually we can let the 'if' statement always go to the parfor branch.
